charts/hobbyfarm/crds/crds.yaml

```diff
@@ -0,0 +1,1318 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: virtualmachines.hobbyfarm.io
+spec:
+  group: hobbyfarm.io
+  names:
+    kind: VirtualMachine
+    plural: virtualmachines
+    singular: virtualmachine
+  preserveUnknownFields: false
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.status
+      name: Status
+      type: string
+    - jsonPath: .status.allocated
+      name: Allocated
+      type: string
+    - jsonPath: .status.public_ip
+      name: publicIP
+      type: string
+    - jsonPath: .status.private_ip
+      name: privateIP
+      type: string
+    name: v1
+    schema:
+      openAPIV3Schema:
+        properties:
+          spec:
+            properties:
+              id:
+                nullable: true
+                type: string
+              keypair_name:
```

Comment:
Since the guacamole changes where introduced, this field was changed to `secret_name`